website/static/js/profile.js

```diff
@@ -693,10 +737,21 @@ var JobViewModel = function() {
     DateMixin.call(self);
     TrackedMixin.call(self);
 
-    self.institution = ko.observable('').extend({required: true, trimmed: true});
     self.department = ko.observable('').extend({trimmed: true});
     self.title = ko.observable('').extend({trimmed: true});
 
+    self.institution = ko.observable('').extend({
+        trimmed: true,
+        required: {
+            onlyIf: function() {
+                if (!!self.department() || !!self.title()) {
```

Comment:
This can be simplified to

```js
return !!self.department() || !!self.title();
```

Remember, these are antipatterns:

```js
if (condition) { return true; }
return condition ? true : false;

// both can be simplified to just
return condition
```